WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/DashboardJetpackSocialCardCell.swift

```diff
@@ -0,0 +1,231 @@
+        let dictionary = repository.dictionary(forKey: Constants.hideNoConnectionViewKey) as? [String: Bool]
+        return dictionary ?? [:]
+    }
+
+    // MARK: - UI Properties
+
+    private var cardTitle: String {
+        switch displayState {
+        // TODO: Out of shares title
+        case .noConnections:
+            return Constants.connectTitle
+        default:
+            return ""
+        }
+    }
+
+    private var cardFrameView: BlogDashboardCardFrameView {
+        let frameView = BlogDashboardCardFrameView()
+        frameView.translatesAutoresizingMaskIntoConstraints = false
+        frameView.setTitle(cardTitle)
+        frameView.onEllipsisButtonTap = { }
+        frameView.ellipsisButton.showsMenuAsPrimaryAction = true
+        frameView.ellipsisButton.menu = contextMenu
+
+        return frameView
+    }
+
+    private var contextMenu: UIMenu {
+        // TODO: Out of shares context menu handler
+        let hideNoConnectionView: UIActionHandler = { [weak self] _ in
+            guard let self else {
+                return
+            }
+
+            self.isNoConnectionViewHidden = true
+            self.dashboardViewController?.reloadCardsLocally()
+        }
+
+        let hideThisAction = UIAction(title: Constants.hideThis,
+                                      image: Constants.hideThisImage,
+                                      attributes: [UIMenuElement.Attributes.destructive],
+                                      handler: hideNoConnectionView)
+        return UIMenu(title: String(), options: .displayInline, children: [hideThisAction])
+    }
+
+    // MARK: - Initializers
+
+    override init(frame: CGRect) {
+        self.repository = UserPersistentStoreFactory.instance()
+        super.init(frame: frame)
+    }
+
+    init(repository: UserPersistentRepository = UserPersistentStoreFactory.instance()) {
+        self.repository = repository
+        super.init(frame: .zero)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - DashboardCollectionViewCell
+
+    func configure(blog: Blog, viewController: BlogDashboardViewController?, apiResponse: BlogDashboardRemoteEntity?) {
+        self.blog = blog
+        self.dashboardViewController = viewController
+        updateDisplayState(for: blog)
+    }
+
+    // MARK: - Functions
+
+    static func shouldShowCard(for blog: Blog) -> Bool {
+        guard FeatureFlag.jetpackSocial.enabled else {
+            return false
+        }
+        // TODO: Show when user is out of shares
+        return showNoConnectionView(for: blog)
+    }
+
+    // MARK: - Constants
+
+    struct Constants {
+        static let hideNoConnectionViewKey = "dashboard-social-no-connection-view-hidden"
+        static let connectTitle = NSLocalizedString("dashboard.card.social.noconnections.title",
+                                                    value: "Connect your social profiles",
```

Comment:
Based on the discussion in p1689934916528639/1689897525.127499-slack-C05362VSXFU, I suppose this should be updated to `Connect social accounts`? 

The discussion suggested that the text should be updated `Connect accounts` instead of `Connect social accounts` because the "Social" part is implied in the pre-publishing sheet (navigation title) and in the post settings (section title). Since there's no hint for "Social" in the dashboard, I think it makes sense to use `Connect social accounts` here as the card title.

```suggestion
                                                    value: "Connect social accounts",
```

Reply:
`Connect social accounts` sounds like it'd be right then. I can double check with everyone just so we're all on the same page though.

Reply:
Strings have been updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/21180/commits/2ee12d597ce70173e1cefe1a9a53e972d7ae71e3.